README.md

```diff
@@ -109,6 +109,26 @@ heat_flux_config:
 
 See [heat_flux_models.py](fehmtk/preprocessors/heat_flux_models.py) for a full list of supported models and their required params.
 
+#### Boundaries config
+Mapping with a single top level key `flow_configs`, with a list of FlowConfigs as a value:
+```yaml
+boundaries_config:
+  flow_configs:
+  - boundary_model:
+      kind: 'open_flow'
+      params:
+        input_fluid_temp_degC: 2
+        aiped_to_volume_ratio: 1.0e-08
+    outside_zones: ['top']
+    material_zones: []  # these can also be omitted if empty (done with outside_zones below)
+  - boundary_model:
+      kind: 'open_flow'
```

Comment:
I struggled a bit with what to call this - we use this as a "constant pressure" boundary, but this is accomplished by allowing open flow into or out of the node to maintain that pressure. Since the pressure isn't specified here, and this creates a flow file used by the flow macro, I thought this name would be more appropriate - but please let me know what you think, it's easy to change.

Reply:
Yes, I agree with naming something with "flow" because this uses the "flow" macro. The FEHM usage is confusing, as this macro also influences temperature, but the old (topbc) name was not exactly right either (we set top temp with hflx, etc). I think using flow makes sense.